package.json

```diff
@@ -12,7 +12,7 @@
   "main": "main.js",
   "scripts": {
     "postinstall": "electron-builder install-app-deps && rimraf node_modules/dtrace-provider",
-    "test": "npm run eslint && npm run test-server && grunt test",
+    "test": "npm run eslint && npm run test-server && grunt test && mocha --recursive test/modules",
```

Comment:
Couldn’t figure out how to run CommonJS module tests in Electron, so I temporarily run them using `mocha` directly. Would love to discuss!